plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/metastore/glue/DeltaLakeGlueMetastoreTableFilterProvider.java

```diff
@@ -40,7 +40,7 @@ public DeltaLakeGlueMetastoreTableFilterProvider(DeltaLakeGlueMetastoreConfig co
     public Predicate<Table> get()
     {
         if (hideNonDeltaLakeTables) {
-            return DeltaLakeGlueMetastoreTableFilterProvider::isDeltaLakeTable;
+            return DefaultGlueMetastoreTableFilterProvider::isDeltaLakeTable;
```

Comment:
`io.trino.plugin.deltalake.metastore.glue.DeltaLakeGlueMetastoreTableFilterProvider#isDeltaLakeTable` is now unused

Reply:
:| i pressed cmd+z too many times...